src/utils/getSortedGoods.ts

```diff
@@ -0,0 +1,36 @@
+import { SortType } from '../types/SortType';
+
+interface SortingFilters {
+  sortField: SortType;
+  isReversed: boolean;
+}
+
+export function getSortedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortingFilters,
+) {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
```

Comment:
The check for `sortField` is unnecessary. As `sortField` is typed as `SortType`, which is an enum, it should always have a value. You can remove the if condition.